src/tests/todo.test.js

```diff
@@ -0,0 +1,17 @@
+const Task = require('../modules/todotask.js');
+const { getFromLocalStorage } = require('../modules/localStorage.js');
+
+const task = new Task();
+const descriptions = ['Go to work', 'Go to School', 'Go to home'];
+
+describe('add list to task in localStorage', () => {
+  test('save to localStorage', () => {
+    task.add(descriptions[0]);
+    task.add(descriptions[1]);
+    expect(JSON.parse(window.localStorage.getItem('tasks')).length).toEqual(2);
+  });
+  test('remove  from localStorage', () => {
+    task.delete(descriptions[0], getFromLocalStorage());
+    expect(JSON.parse(window.localStorage.getItem('tasks')).length).toEqual(1);
+  });
+});
```

Comment:
- Hey, your test is not passing, please consider re-evaluating it to ensure it passes.. :+1:

Reply:
Hello,
Thank you for reviewing our code, when we run tests, it passed. can you provide more information about it? What's wrong with our code?
Thanks.